modules/efk-logging-about-kibana.adoc

```diff
@@ -7,6 +7,7 @@
 
 {product-title} uses Kibana to display the log data collected by Fluentd and indexed by Elasticsearch.
 
-Kibana is a browser-based console interface to view and manipulate your Elasticsearch data through histograms, line graphs, pie charts, heat maps, and built-in geospatial support. 
+Kibana is a browser-based console interface to query, drill down, and visualize your Elasticsearch data through 
```

Comment:
Is "drill-down" Kibana terminology? Otherwise, it sounds a bit jargony.

Reply:
I was asked by a dev to add that phrase.

Reply:
It's your call, but it's not going to translate well, and IBM Style says to "Avoid slang, jargon, humor, sarcasm, colloquialisms, idioms, emoticons (also called smilies), and metaphors."